src/lib/download/http-ledger.ts

```diff
@@ -22,6 +22,7 @@ export async function openHttpLedgerReadStream(context?: ValidationContext): Pro
       'Prefer':           'return=representation',
       'Accept-Encoding':  'br'
     },
+    cache: 'no-store',
```

Comment:
Bonus change that prevents Ketting from storing the ledger in cache.

Reply:
Thanks Evert! I'm glad you found something to fix in Ketting, always good to make improvements in your core library. I wonder if evently is sending back the right cache-control headers in this case though? Would ketting skip storing if cache control was `no-store`? Would we need this explicit setting in the Ketting request then?

Reply:
Ketting will cache basically everything indefinitely unless asked otherwise. It's a long story, but adding cache-control would at least now not make a difference